icu4c/source/test/cintltst/ucptrietest.c

```diff
@@ -1145,6 +1145,9 @@ TrieTestSet3Initial9(void) {
     testTrieRanges("set3-initial-9", FALSE,
         setRanges3, UPRV_LENGTHOF(setRanges3),
         checkRanges3, UPRV_LENGTHOF(checkRanges3));
+    testTrieRanges("set3-initial-9", TRUE,
```

Comment:
Please either just *change* the flag in the existing call, or else add a suffix to the test name (e.g., "set3-initial-9-clone").

Reply:
Done